### ISSUE TITLE
Bounce Keys | Anschlagsverzögerung (Ignore double triggering of the same key [configurable])

### ISSUE THREAD

**Issue by pzauner** [palsoftware/pastiera#221]
Some hardware keyboards may degrade over time or don't have proper integration in the shipped software so they may enter a key twice very rapidly without the user actually triggering the key again. This mainly is a hardware issue that can occur to many keyboards. There is typically some small tolerance between the button physical resistance of the button press and the actual button triggering electric inside and closing the contact. This jitter may result in double keypresses.

To mitigate that behaviour OS manufacturers should account for that or it should be already mitigated in device firmware. However: If this isn't the case we can ofc fix this as an IME as well.

So we'll add this feature as Android doesn't have a native setting for this like proper desktop OS and make it configurable. Users may select via a slider the minimum input delay for the same button to be entered again; under the threshold we simply ignore the input.

Also serving as a accessibilty option. Will be linked there as well or maybe configured there entirely instead of auto substitutions

**Comment by pzauner** [comment]
implemented in 0358987c3f6004ab2f83d571e92a1a4feaeba55a
